addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/blob/s3/S3BlobProvider.java

```diff
@@ -179,4 +189,45 @@ protected String getContentDispositionHeader(Blob blob) {
         return RFC2231.encodeContentDisposition(blob.getFilename(), false, null);
     }
 
+    @Override
+    public BlobStatus getStatus(ManagedBlob blob) throws IOException {
+        String key = stripBlobKeyPrefix(blob.getKey());
+        String objectKey;
+        String versionId;
+        int seppos = key.indexOf(VER_SEP);
+        if (seppos < 0) {
+            objectKey = key;
+            versionId = null;
+        } else {
+            objectKey = key.substring(0, seppos);
+            versionId = key.substring(seppos + 1);
+        }
+        String bucketKey = config.bucketPrefix + objectKey;
+        GetObjectMetadataRequest request = new GetObjectMetadataRequest(config.bucketName, bucketKey);
+        request.setVersionId(versionId);
+        ObjectMetadata metadata;
+        try {
+            metadata = config.amazonS3.getObjectMetadata(request);
+        } catch (AmazonServiceException e) {
+            if (S3BlobStore.isMissingKey(e)) {
+                // don't crash for a missing blob, even though it means the storage is corrupted
+                log.error("Failed to get information on blob: " + key, e);
```

Comment:
can be replaced by log.error("Failed to get information on blob: {}" , key, e); but i think you main logging message are doing using the concatenation string message (i mean on others code part like [here](https://github.com/nuxeo/nuxeo/blob/feature-NXP-28417-glacier-restore-and-status-10.10/addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/blob/s3/S3BlobProvider.java#L76) perhaps we should keep it as it is to more consistency